docker/models/containers.py

```diff
@@ -181,7 +181,8 @@ def exec_run(self, cmd, stdout=True, stderr=True, stdin=False, tty=False,
             user (str): User to execute command as. Default: root
             detach (bool): If true, detach from the exec command.
                 Default: False
-            stream (bool): Stream response data. Default: False
+            stream (bool): Stream response data. Ignored if ``detach`` is true.
+                Default: False
```

Comment:
This change SGTM - we should probably look (maybe from an API perspective) to make this combination an error instead of silently ignore (I recall some recent conversation around a similar case with the Go SDK) /cc @laurazard